linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/CoreJSLib.scala

```diff
@@ -793,18 +801,20 @@ private[emitter] object CoreJSLib {
           })
         })
 
-        defineFunction("doubleToLong", paramList(x), Return {
+        defineFunction("doubleToLong", paramList(x),
+            Return {
           If(x < double(-9223372036854775808.0), { // -2^63
```

Comment:
This also cannot be correct: The nested block has *less* indentation than its parent.